src/plugins/intel_cpu/src/utils/shape_inference/shape_inference.hpp

```diff
@@ -40,6 +41,10 @@ class IStaticShapeInfer : public IShapeInferCommon {
     using IShapeInferCommon::infer;
 
     virtual Result infer(const std::vector<StaticShape>& input_shapes, const ov::ITensorAccessor& tensor_accessor) = 0;
+    Result infer(const std::vector<StaticShape>& input_shapes,
```

Comment:
This should be removed and classes implementing this interface should provide implementation. 
The `ShapeInferBase` provide this implementation (if required can be change).

Reply:
already revert modification of NgraphShapInfer